diff --git a/KEYS b/KEYS
index fa14c61..34f08bd 100644
--- a/KEYS
+++ b/KEYS
@@ -1,5 +1,5 @@
 -----Quitting-----
-escape/q                - quit
+q                       - quit
 Q                       - quit without saving the configuration or changes
 W                       - save changes
 
diff --git a/main.ml b/main.ml
index 9342cda..e2eb5a8 100644
--- a/main.ml
+++ b/main.ml
@@ -4695,7 +4695,7 @@ let viewkeyboard key mask =
      )
      else impmsg "keyboard link navigation does not work under rotation"
 
-  | Escape | Ascii 'q' ->
+  | Ascii 'q' ->
      begin match state.mstate with
      | Mzoomrect _ ->
         resetmstate ();
@@ -5167,7 +5167,7 @@ let viewkeyboard key mask =
      G.postRedisplay "|";
      state.mode <- Textentry (te, onleave);
 
-  | (Ascii _|Fn _|Enter|Left|Right|Code _|Ctrl _) ->
+  | (Escape|Ascii _|Fn _|Enter|Left|Right|Code _|Ctrl _) ->
      vlog "huh? %s" (Wsi.keyname key)
 ;;
 
